elastic/kibana
# [Console] Fix file paths for Windows

## Summary
Fixes https://github.com/elastic/kibana/issues/151032
This PR fixes the paths for autocomplete definitions that are currently not working on Windows, possibly a regression after this dependency update https://github.com/elastic/kibana/pull/138571. 

### How to test
1. Run the command `node scripts/build --all-platforms` in the Kibana repo
2. Copy the created build distro for Windows from the folder `build/default/kibana-8.8.0-SNAPSHOT-windows-x86_64` to your Windows machine (you might need to create a VM in Parallels)
3. In your Windows machine, get an ES snapshot from [here](https://artifacts-api.elastic.co/v1/search/8.8-SNAPSHOT/elasticsearch) and start it with `bin\elasticsearch`
4. Start Kibana with `bin\kibana` in the copied distro folder
5. On the home page, click the button "Try sample data", then expand "Other sample data sets" and add a couple of sample indices here. 
6. Navigate to Dev tools Console and start typing `GET kibana`. You should see a list of indices in autocomplete suggestions.

### Screen recording 

https://user-images.githubusercontent.com/6585477/221226256-1b0b6509-6e52-445e-b4cb-0cffb69bc907.mov
